mule-migration-tool-e2e-tests/src/test/resources/e2e/ftp/ftp/output/src/main/mule/mule-config.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:compatibility="http://www.mulesoft.org/schema/mule/compatibility" xmlns:ftp="http://www.mulesoft.org/schema/mule/ftp" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/ftp http://www.mulesoft.org/schema/mule/ftp/current/mule-ftp.xsd http://www.mulesoft.org/schema/mule/compatibility http://www.mulesoft.org/schema/mule/compatibility/current/mule-compatibility.xsd http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <ftp:config name="ftpSource">
+        <ftp:connection host="localhost" port="${ftpSourcePort}" username="admin" password="admin" passive="true" transferMode="BINARY">

```

Comment:
the previous test is throwing errors saying that FTP is not supported, but this one does not. One of them might be wrong